cypress/e2e/shared/flowsSidebarExportToClipboard.test.ts

```diff
@@ -130,34 +137,37 @@ describe('Flows', () => {
       cy.getByTestID('tree-nav')
 
       cy.clickNavBarItem('nav-item-flows')
+      cy.intercept('PATCH', `/api/v2private/notebooks/*`).as(
+        'NotebooksPatchRequest'
+      )
     })
+  })
 
-    it('Export to Clipboard as Code', () => {
-      const query = 'buckets()'
+  it('Export to Clipboard as Code', () => {
+    const query = 'buckets()'
 
-      createEmptyNotebook()
-      addFluxQueryInNotebook(query)
-      openCopyAs()
+    createEmptyNotebook()
+    addFluxQueryInNotebook(query)
+    openCopyAs()
 
-      cy.get('@org').then(({name}: Organization) => {
-        getClients(name, query).forEach(client => {
-          verifyClientCode(client)
-        })
+    cy.get('@org').then(({name}: Organization) => {
+      getClients(name, query).forEach(client => {
+        verifyClientCode(client)
       })
     })
+  })
 
-    it('Export to Clipboard as Code with Custom Bucket', () => {
-      const bucket = '_customBucket'
-      const query = `from(bucket: "${bucket}"`
+  it('Export to Clipboard as Code with Custom Bucket', () => {
+    const bucket = '_customBucket'
+    const query = `from(bucket: "${bucket}")`
```

Comment:
possibly, i've seen this careless leaving off parens elsewhere, so it might be deliberate to take advantage of autocomplete.

But i was seeing it without the closing parens locally.